src/codewhisperer/models/constants.ts

```diff
@@ -18,6 +18,11 @@ export const CodeWhispererConstants = {
     invocationKeyThreshold: 15,
     specialCharactersList: ['{', '[', '(', ':', '\t', '\n'],
     normalTextChangeRegex: /[A-Za-z0-9]/g,
+    AutoSuggestion: {
+        settingId: 'cwspr_autoSuggestionActivation',
```

Comment:
I know it's verbose but using the exact same 'namespace' with telemetry will help in the long run.
```suggestion
        settingId: 'codewhisperer_autoSuggestionActivation',
```

Reply:
make sense, will update it!